### PR TITLE
According to issue #3553, the default logtimezone is always UTC

### PR DESCRIPTION
**Pull request by robert-scheck**

### CODE REVIEW

**Comment by MorrisJobke** [comment]
Then I would go for remove this setting at all. cc @LukasReschke @nickvergessen Okay?

**Comment by nickvergessen** [comment]
What? No? you can overwrite it in the config file. It's just set to UTC by default. No need to remove the setting. We can remove the `set()` call, yes. But not the config option.

File: lib/private/Setup.php [hunk 1]
```diff
@@ -373,10 +373,8 @@ public function install($options) {
 			Setup::updateHtaccess();
 			Setup::protectDataDirectory();
 
-			//try to write logtimezone
-			if (date_default_timezone_get()) {
-				$config->setSystemValue('logtimezone', date_default_timezone_get());
-			}
+			//try to write logtimezone with UTC as default
+			$config->setSystemValue('logtimezone', 'UTC');
```
Comment (MorrisJobke) [inline lib/private/Setup.php]: Okay. Right - remove this entry just completely. The default is UTC anyways on all other calls.

**Review by MorrisJobke — CHANGES_REQUESTED** [review]
See comment

**Comment by LukasReschke** [comment]
@MorrisJobke @nickvergessen Happier now? :)

**Review by nickvergessen — APPROVED** [review]

**Review by MorrisJobke — APPROVED** [review]

**Pull request merged**